lib/membrane_live/room.ex

```diff
@@ -101,10 +102,10 @@ defmodule MembraneLive.Room do
 
   @impl true
   def handle_call({:remove_peer, peer_id}, _from, state) do
+    state = handle_peer_left(state, peer_id)
+
     case Jellyfish.Room.delete_peer(state.client, state.room.id, peer_id) do
       :ok ->
```

Comment:
So now we will remove the peer from the state even though it can still be on jellyfish?